packages/client/hmi-client/src/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -153,98 +151,66 @@ const stratifyModel = () => {
 };
 
 const resetModel = () => {
-	const kernel = jupyterSession.value?.session?.kernel;
-	if (!kernel || !model.value) return;
-
-	const messageBody = {
-		session: jupyterSession?.value?.name || '',
-		channel: 'shell',
-		content: {},
-		msgType: 'reset_request',
-		msgId: createMessageId('stratify_request')
-	};
-	const contextMessage: JupyterMessage = createMessage(messageBody);
-	kernel.sendJupyterMessage(contextMessage);
+	if (!model.value) return;
+
+	const resetMessage = kernelManager.sendMessage('reset_request', {});
+
+	if (resetMessage) {
+		resetMessage.register('reset_response', handleResetResponse);
+	}
```

Comment:
If this is fluent, why not doing like this?
```suggestion
	kernelManager
	    .sendMessage('reset_request', {})
	    ?.register('reset_response', handleResetResponse);
```

Reply:
changed as suggested